src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder str = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder builder = new StringBuilder();
```
naming should be informative